develop/upload-test-results.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
+
+echo "uploading test results from $(pwd)"
+
+for FILE in *.junit.xml; do
```

Comment:
I'd use a lowercase name for a shell variable (the convention is environment variables are uppercase but this isn't an environment variable)

Reply:
Good point; fixed!